specs/core/0_beacon-chain.md

```diff
@@ -1405,22 +1392,13 @@ def exit_validator(state: BeaconState, index: int) -> None:
     validator = state.validator_registry[index]
 
     # The following updates only occur if not previous exited
-    if validator.exit_slot <= state.slot + ENTRY_EXIT_DELAY:
+    if validator.exit_slot <= state.slot - state.slot % EPOCH_LENGTH + ENTRY_EXIT_DELAY:
```

Comment:
As discussed, we should probably round *up* to the nearest epoch boundary (as opposed to round down).